backend/src/services/payment/payment.services.ts

```diff
@@ -0,0 +1,19 @@
+import { environment } from "../../configs/environment"
+import { stripe } from "../../configs/stripe"
+
+export const paymentService = {
+  createPaymentIntentTHB: async (amount: number) => {
+    if (amount < 10) {
```

Comment:
can you change it to < 0 so that it aligns with the SQL schema?

`Amount INTEGER  CHECK(Amount BETWEEN 0 AND 1000000 ),`

Reply:
Do you want me to change the sql rule now or we gonna modify it later?

Reply:
I think change it now